docs/addons/neuron-monitor-addon.md

```diff
@@ -0,0 +1,432 @@
+# Neuron Monitor Addon
+
+[Neuron Monitor](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/tools/neuron-sys-tools/neuron-monitor-user-guide.html) collects metrics and stats from the Neuron Applications running on the system and streams the collected data to stdout in JSON format.
+
+These metrics and stats are organized into metric groups which can be configured by providing a configuration file as described in Using neuron-monitor
+
+When running, neuron-monitor will:
+
+- Collect the data for the metric groups which, based on the elapsed time since their last update, need to be updated
+- Take the newly collected data and consolidate it into a large report
+- Serialize that report to JSON and stream it to stdout from where it can be consumed by other tools - such as the sample neuron-monitor-cloudwatch.py and neuron-monitor-prometheus.py scripts.
+- Wait until at least one metric group needs to be collected and repeat this flow
+
+## Usage
+
+This example can be found on [cdk-aws-observability-acceletator](https://github.com/aws-observability/cdk-aws-observability-accelerator)
```

Comment:
Please remove this entire Observability Pattern example. This will confuse people. Please add a documentation which shows the usage of this particular addon and remove this rest. I will also remove all these images from this PR because the images are from Grafana which is not a functionality of this addon. Overall maintain the documentation required for this addon and remove the rest. These can be used for Observability accelerator pattern.

Reply:
Adressed